Tella/Data/Networking/Repositories/UwaziServerRepositories.swift

```diff
@@ -139,6 +139,23 @@ class UwaziServerRepository: WebRepository {
             }
             .eraseToAnyPublisher()
         }
+    
+    func getRelationshipEntities(serverURL: String, cookie: String, templatesIds: [String?]) -> AnyPublisher<[UwaziRelationshipList], APIError> {
+        let apiResponse: APIResponse<UwaziRelationshipDTO> = getAPIResponse(endpoint: API.getRelationshipEntities(serverURL:serverURL, cookie:cookie))
+
+        return apiResponse
+            .compactMap{ $0.0 }
+            .map { dto in
+                let shouldFetchAllTemplates = templatesIds.contains { $0?.isEmpty ?? false }
+                return dto.rows
```

Comment:
Let's Simplify the logic to->  `dto in  let shouldFetchAllTemplates = templateIds.isEmpty`